build/root/var/lib/ci-artifacts/entrypoints.d/gpu_operator_run_libs.sh

```diff
@@ -0,0 +1,115 @@
+#! /usr/bin/env bash
+
+set -o pipefail
+set -o errexit
+set -o nounset
+
+prepare_cluster_for_gpu_operator() {
+    source $ENTRY_LIB_DIR/entitle.sh 
```

Comment:
`ENTRY_LIB_DIR` is undefined

my initial idea was to `source $ENTRY_LIB_DIR/*` to avoid such issues
then you would need to include the `switch case` in a function instead of calling the script file

[and in this case remove the `set -o` at the top top of of the files]

could you also remove the double empty lines here and there?

Reply:
hum, the variables is exported in the parent script so it won't crash :heavy_check_mark: 
but I find it too confusing to pass the path like that,
I prefer either a _common.sh script to source in all the libraries (as I do in the toolbox)
or as I suggested above, source everything and call functions